spec.emu

```diff
@@ -74,7 +74,9 @@ contributors: Nicolò Ribaudo
           1. If _P_ is a Symbol, then
             1. Return ! OrdinaryGet(_O_, _P_, _Receiver_).
           1. <ins>Let _m_ be _O_.[[Module]].</ins>
-          1. <ins>If _m_ is not a Cyclic Module Record, or both _m_.[[Status]] is ~linked~ and AnyDependencyNeedsAsyncEvaluation(_m_) is *false*, then</ins>
+          1. <ins>If _m_ is not a Cyclic Module Record, then</ins>
+            1. <ins>Perform ? EvaluateSync(_m_).</ins>
```

Comment:
EvaluateSync unconditionally calls `module.Evaluate()`. It will cause the target non Cyclic Module Record to be evaluated every time its namespace object is accessed.

Reply:
Evaluate() can be called multiple times and it's expected to cache it's result:
> Returns a promise for the evaluation of this module and its dependencies, resolving on successful evaluation or if it has already been evaluated successfully, and rejecting for an evaluation error or if it has already been evaluated unsuccessfully. 

https://tc39.es/ecma262/multipage/ecmascript-language-scripts-and-modules.html#table-abstract-methods-of-module-records

Dynamic import also calls Evaluate() every time, even if a module is already evaluated.